hunts/src/tagFilter.js

```diff
@@ -0,0 +1,33 @@
+function getDescendants(tableRow) {
```

Comment:
Is this used? Ctr+f on my phone doesn't give anything

Reply:
No, it's left over from when it was used. Will delete.